modes.xml

```diff
@@ -11,7 +11,7 @@
 
   <mode name="spa-gener" install="yes">
     <pipeline>
-      <program name="lt-proc -g">
+      <program name="lt-proc -b">
```

Comment:
since we're not extracting caps here, I don't think changing the output format makes sense